doc/interop-test-descriptions.md

```diff
@@ -90,34 +90,42 @@ Client asserts:
 * clients are free to assert that the response payload body contents are zero
   and comparing the entire response message against a golden response
 
-### large_compressed_unary
-
-This test verifies compressed unary calls succeed in sending messages. It
-sends one unary request for every payload type, with and without requesting a
-compressed response from the server.
-
-In all scenarios, whether compression was actually performed is determined by
-the compression bit in the response's message flags.
+### client_compressed_unary
 
+This test verifies the client can compress unary messages. It sends two
+unary requests with their payloads marked as COMPRESSABLE. One request will be
```

Comment:
`response_type: COMPRESSABLE`  doesn't mean unary request will have their payload marked as compressable (as your description suggests). It means we expect the response to be of compressable type - but that's irrelevant for request compression by the client.